src/authInfo.ts

```diff
@@ -24,6 +25,7 @@ import {
 import { createHash, randomBytes } from 'crypto';
 import * as dns from 'dns';
 import { OAuth2, OAuth2Options, TokenResponse } from 'jsforce';
+import { Connection as JSConnection } from 'jsforce';
```

Comment:
I'm wondering if it's possible to merge these imports into one line? I think I've seen this before. I could be wrong. 

_example:_
import { OAuth2, OAuth2Options, TokenResponse, Connection as JSConnection  } from 'jsforce';